DataManagementSystem/DB/FileCatalogComponents/FileManagerBase.py

```diff
@@ -1257,14 +1207,14 @@ def _checkLFNPFNConvention( self, lfn, pfn, se ):
     result = self._getStorageElement( se )
```

Comment:
I still believe that this method should not be here, and even more that we should not instanciate an SE in a catalog